lib/rubocop/cop/style/nested_interpolation.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks nested interpolations
```

Comment:
I think you can expand the documentation of this cop with more rationale and examples that are following the same conventions as those in existing cops. Might be a good idea to link the cop to a style guide entry.

Reply:
@bbatsov I've updated the description a little bit but I'm not sure that this is enough. (But the style should be the same as in existing cops). Also, added one more example.

> Might be a good idea to link the cop to a style guide entry.

Does it mean that we need to add it to the style guide? And can you guide me on how to do it?